cibuild.cmd

```diff
@@ -145,5 +147,15 @@ exit /b 1
 @REM Kill any instances of msbuild.exe to ensure that we never reuse nodes (e.g. if a non-roslyn CI run
 @REM left some floating around).
 
-taskkill /F /IM vbcscompiler.exe 2> nul
-taskkill /F /IM msbuild.exe 2> nul
+@REM An error-level of 1 means that the process was found, but could not be killed.
+taskkill /F /IM vbcscompiler.exe > nul
+if %ERRORLEVEL% == 1 GOTO ActuallyFail
+
+taskkill /F /IM msbuild.exe > nul
+if %ERRORLEVEL% == 1 GOTO ActuallyFail
+
+taskkill /F /IM csc.exe > nul
+if %ERRORLEVEL% == 1 GOTO ActuallyFail
```

Comment:
I don't think we should add csc / vbc to the list here.  Killing vbcscompiler is safe.  Killing msbuild is dubious but there are flaws in their process model that really force our hand here.  Killing vbc / csc is always dangerous and there is no reason we should be doing it.  

If we are concerned about these processes then I think we need to go the other direction: fail the build if csc / vbc are still running in our present session.